### PR TITLE
create structure for bookstore

### PR DESCRIPTION
**Pull request by huzaifakhan0308**
- create a form for adding book
- create a navbar and use a `<Route>` for every page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,90 @@
...
+
+- GitHub: [@huzaifakhan0308](https://github.com/huzaifakhan0308)
+- Twitter: [@huzaifakhan0308](https://twitter.com/home?lang=en)
+- LinkedIn: [huzaifa khan](https://www.linkedin.com/in/huzaifa-khan-938140256/)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Feel free to check [issues page](https://github.com/huzaifakhan0308/book-store/issues).
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (roshan-bajgain) [inline README.md:92]: - [ ] Please make sure to include Future Features, including 1-3 potential features to be added.

**Review by roshan-bajgain — COMMENTED** [review]
Hi @huzaifakhan0308, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no all main headers from the [README template](https://github.com/microverseinc/readme-template). please see comment below.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/book.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      <ul>
+        <li>
+          <span>Title by &quot;Author&quot;  </span>
+          <button type="button">Delete</button>
+        </li>
+      </ul>
+      <Form />
+    </div>
+  );
+}
```
Comment (muneebulrehman) [inline src/components/book.jsx]: As per the project requirements, you should have a `BookList` element that renders a list of books using the `Book` component passing each book to the `Book` component. However, you are directly using the `Book` component. Kindly fix this issue. You don't have to name your components exactly like this, though I would recommend doing so, however, the implementation should be on the same lines.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @huzaifakhan0308 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] I would recommend making your PR a bit more descriptive. You did a lot of things in this milestone. However, you seem to have mentioned two things that you did. Kindly make it a bit more descriptive and including other things that you implemented.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] I would recommend using ES6 arrow functions instead of function declaration way to define functions.
- [ ] As a convention of react, the functional components are generally initialized with a capital letter. However, you are using lower case throughout your application. I would recommend you change it. eg., `const Categories = () => { }` or `const Form = ( )
 => { }`
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @huzaifakhan0308, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- requested changes ✔
- No linter errors ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**